src/commands/mock.ts

```diff
@@ -47,5 +47,31 @@ export const Mock = {
         }
         mockTargets.add(target.id);
         await interaction.reply(`Sure thing, I will begin mocking ${target}!`);
+    },
+
+    Effect: async (message: Message): Promise<void> => {
+
+        let shouldBeLower = true;
+        let length = message.content.length;
+        let userMessage = message.content
+        let mockMessageArray = [...userMessage];
+
+        for (let i = 0; i < length; i++) {
+            if (userMessage[i].toUpperCase() !== userMessage[i].toLowerCase()) {
+                if (shouldBeLower) {
+                    mockMessageArray[i] = userMessage[i].toLowerCase();
+                } else {
+                    mockMessageArray[i] = userMessage[i].toUpperCase();
+                }
+                shouldBeLower = !shouldBeLower;
+            }
+        }
+        let mockMessage = mockMessageArray.join('');
+
+        await message.delete();
```

Comment:
What happens if the message cannot be deleted? This might throw an error which might crash your bot.